components/brave_news/browser/topics_fetcher.cc

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/brave_news/browser/topics_fetcher.h"
+
+#include <string>
+#include <tuple>
+#include <utility>
+#include <variant>
```

Comment:
std::variant is banned, but you don't appear to be using it anyway https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/c++-features.md#std_variant-banned